change/@ni-nimble-components-94a1df54-291a-4470-b64b-45ab1322edf0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "added a design token for graph gridlines",
+  "packageName": "@ni/nimble-components",
+  "email": "vikash.venkatesh@solitontech.local",
```

Comment:
Heads up that this email does end up public in the change log. If you'd prefer to keep it private, you can [follow the instructions in the CONTRIBUTING doc](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file) about using an obfuscated GitHub email instead.

Reply:
Thanks for sharing! I have updated the settings and email address in the change file